packages/discord.js/src/structures/StageChannel.js

```diff
@@ -41,10 +41,10 @@ class StageChannel extends BaseGuildVoiceChannel {
    * Sets a new topic for the guild channel.
    * @param {?string} topic The new topic for the guild channel
    * @param {string} [reason] Reason for changing the guild channel's topic
-   * @returns {Promise<GuildChannel>}
+   * @returns {Promise<StageChannel>}
    * @example
    * // Set a new channel topic
-   * channel.setTopic('needs more rate limiting')
+   * stagechannel.setTopic('needs more rate limiting')
    *   .then(newChannel => console.log(`Channel's new topic is ${newChannel.topic}`))
```

Comment:
Not sure why this is a "new channel":

```suggestion
   *   .then(channel => console.log(`Channel's new topic is ${channel.topic}`))
```